PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate):
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+## Checklist:
```

Comment:
Meh.
`My code follows the code style of this project.` => we have a linter for this.
`My change requires a change to the documentation.` and `I have updated the documentation accordingly` => we only have an API documentation, and this API doc has a linter as well.
`I have read the **CONTRIBUTING** document.` => what are you expecting from this line?
`I have added tests to cover my changes.` => we don't have tests, so this one can be removed for now.
` All new and existing tests passed.` => Travis already does the check.

So I'd remove this check list completely. If you want to list some useful hints maybe write them in the `CONTRIBUTING` file?